modules/ROOT/pages/on-error-scope-concept.adoc

```diff
@@ -60,18 +60,58 @@ to fatal errors, which have error messages that contain the word "fatal":
 ----
 <on-error-continue
   enableNotifications="true" logException="true"
-  doc:name="On Error Continue" type="ANY"
+  type="ANY"
   when='error.cause.message contains "fatal"'/>
 ----
 
 Note that matching conditions are evaluated sequentially, in the order in
-which the On-Error components reside in the Error Handler. So, for example, if
-you want to handle specific HTTP errors in one way but handle all other errors
-a different way, you need to provide an On-Error component configuration for
-the HTTP errors before a component that captures the remaining errors with
-ANY. Keep in mind that default error handling takes place for errors you
```

Comment:
We've lost this section now. I just thought it needed rephrasing, not to be removed entirely as it covers what the default handler will do.

Reply:
Ok, let me know what the issue was with the orig sentence so I can address it: "Keep in mind that default error handling takes place for errors you that you do not explicitly match or use ANY to capture. The default error handlers are configured to produce the handler logs and propagates such unhandled errors."

Reply:
fixed

Reply:
duh! I fixed it. :)